Daytime-Operations/Auxiliary-Telescope/AT-Daytime-Checkout/Daytime Checkout SAL Scripts.rst

```diff
@@ -0,0 +1,93 @@
+.. |author| replace:: *E. Dennihy*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _AT-Daytime-Checkout-Daytime-Checkout:
+
+############################
+Daytime Checkout SAL Scripts
+############################
+
+To ensure the Auxiliary Telesocpe is fully operational and ready for nighttime use, we have created a series of SAL Scripts that successively 
```

Comment:
Apparently, the rule of thumb in writing documentation with lightweight markup languages (like rst) says that we have to create a line break for each new sentence. [Here](https://sembr.org) you can find some reference to that rule (actually, in this link, it talks about "semantic line breaks"). Maybe @patrickingraham can comment on that :)

Reply:
A big ❤️ to semantic line breaks. Works great with LaTeX too!